docs/docs/dev/api.md

```diff
@@ -896,7 +897,7 @@ Manually trigger a new iteration flow.
 
 ---
 
-<span class="request_type">GET</span> ```/workspace/<workspace_id>/force_train?category_name=<my_category_name>```
+<span class="request_type">GET</span> ```/workspace/<workspace_id>/force_train?category_id=<my_category_id>```
```

Comment:
Minor comment: Could you please replace `<my_category_id>` with `<category_id>` to be consistent with the rest of the endpoint documentation?